trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -1284,6 +1326,7 @@ fn process_compact_prestate_common(
 
 // TODO: Move behind a feature flag just used for debugging (but probably not
 // `debug`)...
+#[allow(dead_code)]
 fn parse_just_to_instructions(bytes: Vec<u8>) -> CompactParsingResult<Vec<Instruction>> {
```

Comment:
Do we plan on using it? Otherwise I'd just go for removing it

Reply:
Keep. Debug function.